engine/runtime-instrument-common/src/main/java/org/enso/interpreter/service/ExecutionService.java

```diff
@@ -739,6 +738,10 @@ public static FunctionPointer fromAtomConstructor(AtomConstructor atomConstructo
     }
 
     public static FunctionPointer fromFunction(Function function) {
+      var cons = AtomConstructor.accessorFor(function);
```

Comment:
Introducing the `accessorFor` method allows us to make `QualifiedAccessorNode` package private, @4e6.